### PR TITLE
Issue templates

### PR DESCRIPTION
**Pull request by alorma**
Created two issue templates.

One for bug reports
One for new features / barista methods

### CODE REVIEW

**Comment by alorma** [comment]
@Sloy 
@SmasSive 
@rocboronat 

What do you think about it?

If you like it, I can do the same for PullRequests, with feature / documentation / something?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+**Library Version:**
+
+2.x.x
+  
+**Describe the Bug:**
+
+A clear description of what is the bug is.
+
+**To Reproduce:**
```
Comment (EsteveAguilera) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I think that "Steps to reproduce the bug" is more descriptive

Comment (alorma) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done!

File: .github/ISSUE_TEMPLATE/Feature_request.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+---
+
+*Please consider making a Pull Request if you are capable of doing so.*
+
+**Description what you'd like to happen:**
+
+A clear description if the feature or behavior you'd like implemented.
+
+**Example code of espresso**
```
Comment (EsteveAguilera) [inline .github/ISSUE_TEMPLATE/Feature_request.md]: This sounds like mandatory to me, and not all Barista users should know about espresso... maybe you should ask for "example espresso code if available" or something like that

Comment (alorma) [reply .github/ISSUE_TEMPLATE/Feature_request.md]: Done

**Comment by EsteveAguilera** [comment]
LGTM

**Comment by rocboronat** [comment]
Give us some time for feedback before merging, @alorma ... 👅

Personally I think that this kind of questions make people hesitate to open issues... but I also know that most projects use them.

By the way, just one thing: the commit message, "Issue templates" is not the best one. This post made me think about it some time ago, hope you like it! https://chris.beams.io/posts/git-commit/

**Pull request merged**